src/storage/mutate/DeleteVerticesProcessor.cpp

```diff
@@ -59,11 +59,10 @@ void DeleteVerticesProcessor::process(const cpp2::DeleteVerticesRequest& req) {
         }
     } else {
         callingNum_ = req.parts.size();
-        std::for_each(req.parts.begin(), req.parts.end(), [&](auto &partVerticse) {
-            auto partId = partVerticse.first;
-            const auto &vertices = partVerticse.second;
-            auto atomic = [&]() -> std::string {
-                return deleteVertices(spaceId, partId, vertices);
+        std::for_each(req.parts.begin(), req.parts.end(), [&](auto &pv) {
```

Comment:
I still don't think `&` in closure capture list is a good way, better list the reference capture.

Reply:
Good point